docs/Tutorials/Scaling/Kubernetes/deployingk8s.md

```diff
@@ -88,22 +165,25 @@ Where:
     - **&lt;PASSWORD&gt;** is the password for the Postgres database.
     - **&lt;SERVERURL&gt;** is the loction for the Postgres database.
 
+    !!! warning "Warning - Using an SSL Connection"
+        In this guide, we will disable SSL for ease of setup for local Kubernetes. In a production environment, an SSL connection is highly encouraged for added security and data encryption.
+
```

Comment:
We did the work to enabled SSL on our docker based psql -- that theme should continue and we should leave this as no-verify, and recommend to use either a managed or our docker, and discourage local or simply not provide instructions for it.